examples/tv-casting-app/tv-casting-common/core/CastingPlayer.cpp

```diff
@@ -29,9 +29,7 @@ namespace core {
 
 CastingPlayer * CastingPlayer::mTargetCastingPlayer = nullptr;
 
-// TODO: Verify why commissioningWindowTimeoutSec is a "unsigned long long int" type. Seems too big.
-void CastingPlayer::VerifyOrEstablishConnection(ConnectionCallbacks connectionCallbacks,
-                                                unsigned long long int commissioningWindowTimeoutSec,
+void CastingPlayer::VerifyOrEstablishConnection(ConnectionCallbacks connectionCallbacks, uint64_t commissioningWindowTimeoutSec,
```

Comment:
comment was removed ... so why are u32 seconds not enough (2^32 seconds is a bit over 136 years ...)

Reply:
For prior art, in the chip specification, the `ComissioningTimeout` for `OpenCommissioningWindow` is a u16 (i.e. up to about 18 hours)

Reply:
Yes, I confirmed with sharadb-amazon Sharad Binjola that uint16_t/18 hours is enough for a ComissioningTimeout. Not sure why we had such a big type before. Thanks!